packages/legend-query-builder/src/components/__tests__/QueryBuilderFunctionExplorerTree.test.tsx

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DependencyManager, stub_RawLambda } from '@finos/legend-graph';
+import { integrationTest } from '@finos/legend-shared/test';
+import TEST_DATA_SimpleCalendarModel from '../../stores/__tests__/TEST_DATA__QueryBuilder_Model_Calendar.json' with { type: 'json' };
+import { expect, test } from '@jest/globals';
+import {
+  findByText,
+  fireEvent,
+  getByText,
+  getByTitle,
+  waitFor,
+} from '@testing-library/dom';
+import { act } from 'react';
+import { TEST_DATA__ModelCoverageAnalysisResult_Calendar } from '../../stores/__tests__/TEST_DATA__QueryBuilder_Calendar.js';
+import { QUERY_BUILDER_TEST_ID } from '../../__lib__/QueryBuilderTesting.js';
+import {
+  dragAndDrop,
+  TEST__setUpQueryBuilder,
+} from '../__test-utils__/QueryBuilderComponentTestUtils.js';
+import type { EntitiesWithOrigin } from '@finos/legend-storage';
+
+const firstDependencyEntities = {
```

Comment:
lets enhance this example. lets include functions with params (both primitive and complex classes)